user snippets/elementRightClick.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Simulates right-clicking an element to display a context menu.
+ * Customize by adding a CSS selector to the element variable.
+ * To use an xPath, integrate the getElementByXpath snippet.
```

Comment:
might be helpful to include a function at the bottom for the `getElementByXpath` snippet so a user doesn't have to go looking for it.